### ISSUE TITLE
Not respecting :timestamp-opts

### ISSUE THREAD

**Issue by MiyamotoAkira** [viesti/timbre-json-appender#24]
When I setup code like

```
(defn init []
  (timbre/merge-config!
   {:timestamp-opts
    {:pattern  "yyyy-MM-dd'T'HH:mm:ss.SSSX"
     :locale   :jvm-default
     :timezone :utc         #_(java.util.TimeZone/getTimeZone "Europe/Amsterdam")}})
  (tas/install))
  ```
(or with the `merge-config!` after `tas/install`)

The appender doesn't seems to respect the changes for the timestamp

**Comment by MiyamotoAkira** [comment]
The issue is caused by using instant, instead of timestamp_ when preparing the data to be outputted.

**Comment by viesti** [comment]
Thanks for the issue and the fix! :)

**Comment by viesti** [comment]
Released the change as 0.2.5, thank you! :)
